websites/P/Polywork/presence.ts

```diff
@@ -0,0 +1,56 @@
+const presence = new Presence({
+		clientId: "1021098794894499952",
+	}),
+
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+	presence.on("UpdateData", async () => {
+		const presenceData: PresenceData = {
+				largeImageKey: "polywork",
+				details: "Browsing..",
+				startTimestamp: browsingTimestamp,
+			}
+		
+			if (document.location.hostname == "polywork.com") {
```

Comment:
```suggestion
			if (document.location.hostname === "polywork.com") {
```

Reply:
Actually, you can completely remove this if statement altogether, as your presence is only running on one hostname...